service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java

```diff
@@ -47,44 +52,25 @@ private Date convertToDate(OffsetDateTime submissionTimestamp) {
     return null;
   }
 
-  private RunLog convertToRunLog(WorkflowQueryResult queryResult) {
-    // Note: Cromwell's /query endpoint doesn't return 'workflowUrl' or 'workflowInputs' hence
-    // Setting it 'null' for now
+  private RunLog runToRunLog(Run run) {
+
     return new RunLog()
-        .runId(queryResult.getId())
-        .state(convertToRunState(queryResult.getStatus()))
-        .workflowUrl(null)
-        .name(queryResult.getName())
+        .runId(run.id().toString())
+        .workflowUrl(run.runSet().method().methodUrl())
+        .name(run.entityId())
```

Comment:
Do we store workflow/run name in database?

Reply:
Entity_id was the closest thing I found to storing a name. I didn't find anything specifically identified as 'name' in the database.

Reply:
I guess then for now it can return `null` and we can populate it later on when we have the name. What do you think?

Reply:
This makes sense to me